company.el

```diff
@@ -645,16 +645,35 @@ commands in the `company-' namespace, abort completion."
                         (repeat :tag "Commands" function))
                  (repeat :tag "Commands" function)))
 
-(defcustom company-show-numbers nil
-  "If enabled, show quick-access numbers for the first ten candidates."
+(define-obsolete-variable-alias
+ 'company-show-numbers
+ 'company-show-quick-access
+ "0.10.0")
```

Comment:
The next version will hopefully be 1.0 :slightly_smiling_face: I can change this line later either way.

Not sure if mentioning an unreleased version in an obsoletion message is idiomatic, but it's probably okay.

Reply:
Great! I'm all for it (with a preference of a semantic versioning).

> unreleased version

It seemed quite idiomatic to me. **What would be an alternative?** "0.9.13" seems vague, because the release doesn't contain the obsoletion definitions. (And that that the argument *WHEN* isn't optional since Emacs 28.1. BTW, the day before I re-compiled Emacs from master: the app shows up as 28.0.50 and contains the features marked for unreleased 28.1.)

Reply:
Dmitry, I only now accidentally (consciously?) noticed there's a couple more of "0.9.14" version appearances in the code. They'd require a fix up too.

Reply:
Good eye. :-)

I suppose you could also use the "0.9.14" version string, which will then have to be changed wholesale before the release.

Reply:
Sounds good.